packages/gestalt-codemods/41.0.0/__testfixtures__/rename-dropdown-dangerouslyRemoveLayer-to-isWithinFixed.input.js

```diff
@@ -0,0 +1,18 @@
+// @flow strict
+import { Box, Dropdown } from 'gestalt';
+
+export default function TestBox() {
+  return (
+    <Box>
+      <Dropdown id="test-id" onDismiss={() => {}}>
+        <Dropdown.Item onSelect={() => {}} option={{ label: 'Test Option', value: 0 }} />
+      </Dropdown>
+      <Dropdown id="test-id" onDismiss={() => {}}>
+        <Dropdown.Item onSelect={() => {}} option={{ label: 'Test Option', value: 0 }} />
+      </Dropdown>
+      <Dropdown id="test-id" onDismiss={() => {}} isWithinFixed>
```

Comment:
The target prop shouldn't be in the test input, no? Shouldn't this use the old prop name?

Reply:
Don't know why this happened. But yeah, fixed.